content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -954,6 +954,7 @@ const user = await prisma.user.update({
 #### Remarks
 
 - To perform arithmetic operations on update (add, subtract, multiply, divide), use [atomic updates](#atomic-number-operations) to prevent race conditions.
+- If two or more upsert operations happen at the same time, a race condition might happen. As a result, one or more of the upsert operations might throw a unique key constraint error. Your application code can trap this error and retry the operation. [Learn more](#unique-key-constraint-errors-on-upserts).
```

Comment:
The race condition happens if the row doesn't exist. So maybe it should be:
`If two or more upsert operations run at the same time and the row doesn't exist a race condition could happen.`

Reply:
```suggestion
- If two or more upsert operations happen at the same time and the row doesn't already exist, then a race condition might happen. As a result, one or more of the upsert operations might throw a unique key constraint error. Your application code can trap this error and retry the operation. [Learn more](#unique-key-constraint-errors-on-upserts).
```